### PR TITLE
[tests] Fix relative paths in mmptest and make cloned XM nunit projects work properly.

### PR DESCRIPTION
**Pull request by rolfbjarne**
Fixes this build problem with mmptest:

    /Users/builder/jenkins/workspace/xamarin-macios-pr-builder/tests/mmptest/CustomBuildActions.targets: error : Command 'make bin/SimpleClassDylib.dylib bin/SimpleClassStatic.a bin/Mobile-static/MobileBinding.dll' exited with code: 255.

Fixes https://github.com/xamarin/maccore/issues/633.

### CODE REVIEW

File: tools/common/Application.cs [hunk 1]
```diff
@@ -64,7 +64,12 @@ public partial class Application
 		public bool EnableSGenConc;
 		public MarshalObjectiveCExceptionMode MarshalObjectiveCExceptions;
 		public MarshalManagedExceptionMode MarshalManagedExceptions;
-		public bool IsDefaultMarshalManagedExceptionMode;
+
+		bool is_default_marshal_managed_exception_mode;
+		public bool IsDefaultMarshalManagedExceptionMode {
+			get { return is_default_marshal_managed_exception_mode || MarshalManagedExceptions == MarshalManagedExceptionMode.Default; }
+			set { is_default_marshal_managed_exception_mode = value; }
+		}
```
Comment (spouliot) [inline tools/common/Application.cs]: is that change needed ?
it seems new and not used elsewhere in the diff

Comment (rolfbjarne) [reply tools/common/Application.cs]: Yes, it's explained here: e963395a15d8ffcd6deaeac1b70a0d2f1dadcd51

It could be a different PR though (fixing the test build made a test failure show up too, but they're not directly related).

Comment (spouliot) [reply tools/common/Application.cs]: 👍 I missed there was a 2nd commit (and it's log)

**Comment by rolfbjarne** [comment]
Test failure is unrelated (https://github.com/xamarin/maccore/issues/636)
